### PR TITLE
stop specifying deleteMfa body and header params

### LINKED ISSUE #11: deleteMfa operation does not actually require an mfaBody
The mgmt spec says I need to send an mfaBody when deleting the mfa on my identity, but the server responds OK if I don't sent the body.

### PR DESCRIPTION
**Pull request by qrkourier**
closes #11

### CODE REVIEW

**Review by plorenz — CHANGES_REQUESTED** [review]
Change needs to be made in `source/shared/current-identity-mfa.yml` and then the source should be regenerated.

**Comment by qrkourier** [comment]
@plorenz I see. This hunk of the spec is organized under "shared," and that's why the client-only params are present in the management spec. I'll rewrite the associated issue to reflect this, and so we can discuss the best solution, if any.

File: management.yml [hunk 1]
```diff
@@ -3495,15 +3495,6 @@ paths:
       - MFA
       summary: Disable MFA for the current identity
       operationId: deleteMfa
-      parameters:
-      - description: An MFA validation request
-        name: mfaValidation
-        in: body
-        schema:
-          $ref: '#/definitions/mfaCode'
-      - type: string
```
Comment (andrewpmartinez) [inline management.yml:3504]: The header value is required to remove MFA from the current identity. The body was optional, and technically against the HTTP spec, and can be removed. However the header must stay.

**Comment by qrkourier** [comment]
> Change needs to be made in `source/shared/current-identity-mfa.yml` and then the source should be regenerated.

I'm troubleshooting a problem with `go-swagger` invoked by `generate_rest.sh`.

```bash
❯ bash -ex ./scripts/generate_rest.sh                                                                                                                                                                                                                                    
+ command -v swagger                                                                                                                                                                                                                                                     
++ realpath ./scripts/generate_rest.sh                                                                                                                                                                                                                                   
+ scriptPath=/home/kbingham/Sites/netfoundry/github/edge-api/scripts/generate_rest.sh                                                                                                                                                                                    
++ dirname /home/kbingham/Sites/netfoundry/github/edge-api/scripts/generate_rest.sh                                                                                                                                                                                      
+ scriptDir=/home/kbingham/Sites/netfoundry/github/edge-api/scripts                                                                                                                                                                                                      
++ realpath /home/kbingham/Sites/netfoundry/github/edge-api/scripts/..                                                                                                                                                                                                   
+ rootDir=/home/kbingham/Sites/netfoundry/github/edge-api                                                                                                                                                                                                                
++ realpath /home/kbingham/Sites/netfoundry/github/edge-api/source/client.yml                                                                                                                                                                                            
+ clientSourceSpec=/home/kbingham/Sites/netfoundry/github/edge-api/source/client.yml                                                                                                                                                                                     
++ realpath -m /home/kbingham/Sites/netfoundry/github/edge-api/client.yml                                                                                                                                                                                                
+ clientSwagSpec=/home/kbingham/Sites/netfoundry/github/edge-api/client.yml                                                                                                                                                                                              
++ realpath /home/kbingham/Sites/netfoundry/github/edge-api/source/management.yml                                                                                                                                                                                        
+ managementSourceSpec=/home/kbingham/Sites/netfoundry/github/edge-api/source/management.yml                                                                                                                                                                             
++ realpath -m /home/kbingham/Sites/netfoundry/github/edge-api/management.yml                                                       
+ managementSwagSpec=/home/kbingham/Sites/netfoundry/github/edge-api/management.yml                                                 
++ realpath /home/kbingham/Sites/netfoundry/github/edge-api/scripts/template.copyright.txt                                                                                                                                                                               
+ copyrightFile=/home/kbingham/Sites/netfoundry/github/edge-api/scripts/template.copyright.txt                                                                                                                                                                           
+ echo '...flattening client spec'                                
...flattening client spec                                         
+ echo swagger flatten /home/kbingham/Sites/netfoundry/github/edge-api/source/client.yml -o /home/kbingham/Sites/netfoundry/github/edge-api/client.yml --format yaml                                                                                                     
swagger flatten /home/kbingham/Sites/netfoundry/github/edge-api/source/client.yml -o /home/kbingham/Sites/netfoundry/github/edge-api/client.yml --format yaml                                                                                                            
+ swagger flatten /home/kbingham/Sites/netfoundry/github/edge-api/source/client.yml -o /home/kbingham/Sites/netfoundry/github/edge-api/client.yml --format yaml                                                                                                          
2023/03/28 14:36:34 format =  yaml                                
panic: runtime error: invalid memory address or nil pointer dereference [recovered]                                                 
        panic: runtime error: invalid memory address or nil pointer dereference                                                     
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x74fd71]                                                              

goroutine 1 [running]:                                            
gopkg.in/yaml%2ev3.handleErr(0xc00056b8d0)                        
        /home/runner/go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/yaml.go:294 +0x6d                                                           
panic({0xc78e00, 0x13d1740})                                      
        /opt/hostedtoolcache/go/1.19.1/x64/src/runtime/panic.go:884 +0x212                                                          
gopkg.in/yaml%2ev3.(*encoder).node(0xc000601800?, 0xc00031a000?, {0x0?, 0x0?})                                                      
        /home/runner/go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/encode.go:434 +0x31                                                         
gopkg.in/yaml%2ev3.(*encoder).node(0xc000601800, 0xc0002994a0, {0x0, 0x0})                                                          
        /home/runner/go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/encode.go:526 +0x11e5                                                       
gopkg.in/yaml%2ev3.(*encoder).node(0xc000601800, 0xc000299360, {0x0, 0x0})                                                          
        /home/runner/go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/encode.go:526 +0x11e5                                                       
gopkg.in/yaml%2ev3.(*encoder).node(0xc000601800, 0xc000299220, {0x0, 0x0})                                                          
        /home/runner/go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/encode.go:526 +0x11e5                                                       
gopkg.in/yaml%2ev3.(*encoder).node(0xc000601800, 0xc000298960, {0x0, 0x0})                                                          
        /home/runner/go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/encode.go:526 +0x11e5                                                       
gopkg.in/yaml%2ev3.(*encoder).node(0xc000601800, 0xc0000010e0, {0x0, 0x0})                                                          
        /home/runner/go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/encode.go:526 +0x11e5                                                       
gopkg.in/yaml%2ev3.(*encoder).node(0xc000601800, 0xc000000fa0, {0x0, 0x0})                                                          
        /home/runner/go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/encode.go:526 +0x11e5                                                       
gopkg.in/yaml%2ev3.(*encoder).node(0xc000601800, 0xc000000e60, {0x0, 0x0})                                                          
        /home/runner/go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/encode.go:526 +0x11e5                                                       
gopkg.in/yaml%2ev3.(*encoder).node(0xc000601800, 0xc000000dc0, {0x0, 0x0})                                                          
        /home/runner/go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/encode.go:526 +0x11e5                                                       
gopkg.in/yaml%2ev3.(*encoder).node(0xc000601800, 0xc0002d3400, {0x0, 0x0})                                                          
        /home/runner/go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/encode.go:526 +0x11e5                                                       
gopkg.in/yaml%2ev3.(*encoder).node(0xc000601800, 0xc000532000, {0x0, 0x0})                                                          
        /home/runner/go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/encode.go:477 +0x1685                                                       
gopkg.in/yaml%2ev3.(*encoder).nodev(0xcd8a40?, {0xcd8a40?, 0xc000532000?, 0x1?})                                                    
        /home/runner/go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/encode.go:429 +0x53                                                         
gopkg.in/yaml%2ev3.(*encoder).marshalDoc(0xc000601800, {0x0, 0x0}, {0xcd8a40?, 0xc000532000?, 0x0?})                                                                                                                                                                     
        /home/runner/go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/encode.go:101 +0xc5                                                         
gopkg.in/yaml%2ev3.Marshal({0xcd8a40?, 0xc000532000})                                                                               
        /home/runner/go/pkg/mod/gopkg.in/yaml.v3@v3.0.1/yaml.go:222 +0x389                                                          
github.com/go-openapi/swag.JSONMapSlice.MarshalYAML({0xc0000f6600, 0xa, 0x1a000?})                                                  
        /home/runner/go/pkg/mod/github.com/go-openapi/swag@v0.22.3/yaml.go:245 +0x131                                               
github.com/go-swagger/go-swagger/cmd/swagger/commands.writeToFile(0xc0000a9c70?, 0x1, {0x7ffec4e4984d?, 0x41?}, {0x7ffec4e49809, 0x3a})                                                                                                                                  
        /home/runner/work/go-swagger/go-swagger/cmd/swagger/commands/expand.go:65 +0x1ba                                                                                                                                                                                 
github.com/go-swagger/go-swagger/cmd/swagger/commands.(*FlattenSpec).Execute(0xc000530aa0, {0xc000050a20?, 0xc000530aa0?, 0x6?})                                                                                                                                         
        /home/runner/work/go-swagger/go-swagger/cmd/swagger/commands/flatten.go:47 +0x186                                                                                                                                                                                
github.com/jessevdk/go-flags.(*Parser).ParseArgs(0xc0001929a0, {0xc000034080, 0x6, 0x6})                                                                                                                                                                                 
        /home/runner/go/pkg/mod/github.com/jessevdk/go-flags@v1.5.0/parser.go:335 +0x92e                                                                                                                                                                                 
github.com/jessevdk/go-flags.(*Parser).Parse(...)                                                                                   
        /home/runner/go/pkg/mod/github.com/jessevdk/go-flags@v1.5.0/parser.go:191                                                   
main.main()                                                       
        /home/runner/work/go-swagger/go-swagger/cmd/swagger/swagger.go:140 +0x971                                                   

edge-api no-params-delete-mfa*​                                    
❯ swagger version                                                 
version: v0.30.3                                                  
commit: ecf6f05b6ecc1b1725c8569534f133fa27e9de6b                                                                                    

edge-api no-params-delete-mfa*​                                    
❯ which swagger                                                   
/home/linuxbrew/.linuxbrew/bin/swagger                            
```

**Comment by qrkourier** [comment]
Same issue with `go-swagger` CLI v0.30.4.

```bash
❯ alias swagger='podman run --rm -it  --user $(id -u):$(id -g) -e GOPATH=$(go env GOPATH):/go -v $HOME:$HOME -w $(pwd) quay.io/goswagger/swagger:0.30.4'

❯ swagger version                                                                                                                                       
version: v0.30.4
commit: df6da9b77aa9751f06bedb17fcf92b1ab67a7a47
```

**Comment by qrkourier** [comment]
The YAML in the edited fragment seems to be valid.

```bash
$ yq < source/shared/current-identity-mfa.yml >/dev/null && echo YUMMY      
YUMMY
```

**Comment by qrkourier** [comment]
I'll re-generate this branch as spec v0.25.12.

**Review by andrewpmartinez — APPROVED** [review]

**Comment by qrkourier** [comment]
resolve conflicts and chang base to twig this .12 off of the .11 branch

**Pull request merged**